modules/notification/ui/ui.go

```diff
@@ -29,19 +31,24 @@ var (
 
 // NewNotifier create a new notificationService notifier
 func NewNotifier() base.Notifier {
```

Comment:
I think this change could be another smaller PR which should be easier to review.

Reply:
the single commit ebdebd77921ae49102fd613f2a00b6e514a35e62 has been extracted as #10281